contrib/sbase/patches/format-specifiers.patch

```diff
@@ -0,0 +1,140 @@
+Index: sbase/sbase/cal.c
+===================================================================
+--- sbase.orig/sbase/cal.c	2022-12-07 22:20:25.006463307 +0100
++++ sbase/sbase/cal.c	2022-12-08 00:50:50.422832055 +0100
+@@ -111,7 +111,7 @@
+ 				cur_month -= 12;
+ 				cur_year += 1;
+ 			}
+-			snprintf(month_year, sizeof(month_year), "%s %zu", smon[cur_month], cur_year);
++			snprintf(month_year, sizeof(month_year), "%s %lu", smon[cur_month], cur_year);
```

Comment:
`%zu` is a perfectly correct format specifier. Perhaps we have a problem with our `size_t` definition? After all in NetBSD it's defined as `__SIZE_TYPE__` (through `_BSD_SIZE_T_` macro) and not as `unsigned long` (https://mimiker.ii.uni.wroc.pl/source/xref/mimiker/include/stddef.h#5)

Reply:
`size_t` and `ssize_t` issues have been fixed and `format-specifiers.patch` has been removed